### PR TITLE
Basic architecture

### PR DESCRIPTION
**Pull request by arash8877**
added page routes to the front.
made folder 'components' include:
header
searchBar
....

### CODE REVIEW

**Review by YurijPaharev — CHANGES_REQUESTED** [review]
Please note that your changes should not be merged, until other PRs are merged now. You have all your colleagues changes in your branch, probably you should have created separate branches and make PRs from them

**Comment by arash8877** [comment]
Hej Yurii

Thanks for the feedback.
My PR include codes from other team-mate because I pull Kristina's code in
my branch. And it was a mistake 🙈
I am not sure what I should do?!

Kr
Arash

**Comment by YurijPaharev** [comment]
I would suggest to wait until other PRs would be merged, cause then all those commits would be in dev branch an you will be able to see the changes only of your current branch

**Comment by arash8877** [comment]
OK, I will do that. Thanks.
What about my current PR. Do you accept that?

**Comment by YurijPaharev** [comment]
I will look at it when it will be updated with other PR, after merging it to dev. So now you should wait until another PR is approved, push latest updates to your branch and then I will review it

**Comment by arash8877** [comment]
Oh thank you 😊 so much

File: frontend/src/App.tsx
Comment (YurijPaharev) [inline frontend/src/App.tsx]: Excuse me, but why are you deleting the App.tsx file?

**Review by YurijPaharev — APPROVED** [review]

**Pull request merged**
